src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -371,10 +385,9 @@ public void EmitBindingCompleteMessage()
             exclusionSettingsStorage ??= Mock.Of<IExclusionSettingsStorage>();
             logger ??= new TestLogger(logToConsole: true);
             languagesToBind ??= Language.KnownLanguages;
+            solutionBindingOperation ??= Mock.Of<ISolutionBindingOperation>();
 
-            var slnBindOperation = new SolutionBindingOperation();
```

Comment:
The test was using the real type rather than a mock so I changed it to a mock.
Having done that, none of tests failed, so I realised there must be a missing test case.